R/ab_scenario.R

```diff
@@ -234,7 +234,7 @@ ab_save = function(x, f) {
 #' time_afternoon = ab_time_normal(hr = 17, sd = 0.75)
 #' as.POSIXct(trunc(Sys.time(), units="days") + time_afternoon)
 ab_time_normal = function(hr = 8.5, sd = 0.5, n = 1) {
-  round(stats::rnorm(n = n, mean = hr * 60^2, sd = sd * 60^2))
+  round(stats::rnorm(n = n, mean = hr * 60^2, sd = sd * 60^2)) * 10000.0
```

Comment:
This certainly gets the job done, but it breaks the documentation / API of `ab_time_normal`.

Why not make the change during serialization, at https://github.com/a-b-street/abstr/blob/0eb979d5c9d52ea1dce7f3c2aa01eeecfd389b4e/R/ab_scenario.R#L185? Semantically, we want to work in units of seconds everywhere, and just encode this special way in JSON.

Reply:
Good point Dustin, yes it makes sense to do this at the last possible moment, keeping them in more manageable units of seconds (not 10s of milliseconds?) while in memory.

Reply:
Plan to look at this now...